rbac/management/auditLogs/model.py

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
I would prefer name of `audit_log` as we use singular form in project for models